src/DefaultRESTClient.ts

```diff
@@ -183,6 +183,8 @@ export default class DefaultRESTClient<RT, ERT> implements IRESTClient<RT, ERT>
             body: this.body as BodyInit,
             // @ts-ignore (Credentials are not supported on NodeJS)
             credentials: this.credentials,
+            // 60-second timeout expressed in ms
+            timeout: 60000,
```

Comment:
Any reason not to use `signal: AbortSignal.timeout( 60000 )` instead, since the [options doc](https://github.com/node-fetch/node-fetch/tree/2.x?tab=readme-ov-file#options) says `Signal is recommended instead.`

Reply:
If we want to use a timeout, the timeout value should be set on the class and referenced just like we do for credentials. 

So I would suggest we :
- add a new field for timeout, we can default to 60 seconds if we want.
- add new `with*` method so this can be set using the builder pattern. 

Agree with @mooreds - sounds like the API doc suggest using `Signal` instead. 

Then, to complete this work, we'll need to expose this in the FusionAuth REST client. Perhaps we can add this to the builder once this PR is complete. 
- https://github.com/FusionAuth/fusionauth-typescript-client/blob/master/src/FusionAuthClient.ts
- https://github.com/FusionAuth/fusionauth-client-builder/blob/master/src/main/client/typescript.client.ftl

Reply:
I've updated the PR to add a `withAbortSignal()` instead. 

I'm not sure how this gets incorporated into the two files mentioned above, though.